### DISCUSSION TITLE
Bucket List project completed with Quasar

### DISCUSSION THREAD

**Discussion by ccosse** [quasarframework/quasar#15771] [Show and Tell]
Greetings and much gratitude for Quasar.  I made this [Reading Trainer](https://www.readingedge.org) with the latest Quasar.  It is something which I know from personal experience with my own 2 kids to be very effective at training to read carefully and understand.  Looking forward to using Quasar forever ...

**Comment by suleimanas** [comment]
Great work I'm trying to make my kids read more that looks helpful & fun. Thanks
